psw/ae/aesm_service/source/core/ipc/UnixCommunicationSocket.cpp

```diff
@@ -215,7 +215,7 @@ bool UnixCommunicationSocket::init()
         memset(&serv_addr, 0, sizeof(struct sockaddr_un));
         serv_addr.sun_family = AF_UNIX;
         memset(serv_addr.sun_path, 0, sizeof(serv_addr.sun_path));
-        strncpy(serv_addr.sun_path, mSocketBase, sizeof(serv_addr.sun_path));
+        strncpy(serv_addr.sun_path, mSocketBase, sizeof(serv_addr.sun_path)-1);
```

Comment:
This is not directly to the merge request but valgrind noticed this potential buffer overflow.